### DISCUSSION TITLE
Paper node has over 5000 authors

### DISCUSSION THREAD

**Discussion by luishengjie** [snap-stanford/ogb#187] [MAG240M-LSC]
It seems that there are instances where a paper has > 5000 authors. For instance paper node id: 70650791. 107118834, 59591 all have more than 5000 authors. Just to clarify, should this be the case?
Code snippet to generate number of author for each paper:
```
edge_index_writes = dataset.edge_index('author', 'paper')
df_edge = pd.DataFrame(edge_index_writes, columns=['author', 'paper'])
df_edge = df_edge.groupby(['paper']).agg({'author': 'nunique'})
df_edge.reset_index(inplace=True)
print(f"Min: {df_edge_paper['author'].min()}, Max: {df_edge_paper['author'].max()}")
```
Source: https://ogb.stanford.edu/kddcup2021/mag240m/

Thanks
Sheng Jie

**Comment by weihua916** [comment]
Hi! It's an interesting question.

I looked into the original papers, and those papers do contain more than 5000 authors :)

**Reply by luishengjie** [reply]
I see! Are the edges between authors and paper ordered? For instance, for the paper node id 70650791, will the first author id correspond to the first author?

**Reply by weihua916** [reply]
Not ordered. We did not include author ordering information for simplicity.
